src/plugins/advanced_settings/public/management_app/components/form/form.tsx

```diff
@@ -27,30 +28,184 @@ import {
   EuiPanel,
   EuiSpacer,
   EuiText,
+  EuiTextColor,
+  EuiBottomBar,
+  EuiButton,
+  EuiToolTip,
+  EuiButtonEmpty,
 } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { isEmpty } from 'lodash';
+import { i18n } from '@kbn/i18n';
 import { DocLinksStart, ToastsStart } from '../../../../../../core/public';
 
 import { getCategoryName } from '../../lib';
-import { Field } from '../field';
-import { FieldSetting } from '../../types';
+import { Field, getEditableValue } from '../field';
+import { FieldSetting, SettingsChanges, FormState, FieldState } from '../../types';
 
 type Category = string;
+const NAV_IS_LOCKED_KEY = 'core.chrome.isLocked';
```

Comment:
There actually is an API for this though it is a _slightly_ different value, but I think it's actually what you want: `core.chrome.getIsCollapsed$()`. This is slightly different in that this value updates even when the chrome expands/collapses, but not _locked_ (on hover).

The ideal solution here is to allow rendering EuiBottomBar into a different portal container (https://github.com/elastic/eui/issues/1877) and/or provide a Core API for showing the bottom bar UI. Until we have that though, this will work.